src/payment/data/sagas.js

```diff
@@ -269,6 +270,11 @@ export function* handleSubmitPayment({ payload }) {
   }
 }
 
+export function* handleIssueError() {
+  // Show generic <FallbackErrorMessage>
+  yield call(handleErrors, {}, true);
```

Comment:
This will fall under the fallback error because of the `{}` instead of passing in the actual error?

Reply:
Correct!

Reply:
Correct! This is shown here: https://github.com/openedx/frontend-app-payment/blob/c11933ca0e7224802867d6e673482db8ab0d10e9/src/feedback/data/sagas.js#L12